### PR TITLE
GIX-1960: Upgrade next dependencies

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Use new field for NF enhancements.

In this PR, upgrade ic-js dependencies.

# Changes

## Automatic changes

* In package.lock.json with `npm run upgrade:next`.

## Manual changes

* Fix mocks and converters with new fields.

# Tests

* Add new field in expected value of the sns aggregator converters.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd please review

**Review by dskloetd — APPROVED** [review]

**Comment by lmuntaner** [comment]
@dskloetd I did quite some changes because a new fields was missing in quite a few mocks. We need a helper for that mock. Maybe in another PR.

File: frontend/src/tests/lib/pages/ProjectDetail.spec.ts [hunk 1]
```diff
@@ -559,7 +561,7 @@ sale_buyer_count ${saleBuyerCount} 1677707139456
       });
 
       it("should participate without user interaction if there is an open ticket.", async () => {
-        const initialCommitment = { icp: [] };
+        const initialCommitment = { icp: [], has_created_neuron_recipes: [] };
```
Comment (dskloetd) [inline frontend/src/tests/lib/pages/ProjectDetail.spec.ts:564]: What does `has_created_neuron_recipes` have to do with "commitment"?
What are neuron recipes?

Comment (lmuntaner) [reply frontend/src/tests/lib/pages/ProjectDetail.spec.ts:564]: Good question... 🤷

**Pull request merged**